applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -194,6 +194,8 @@ KRATOS_CREATE_VARIABLE(Matrix, PLASTIC_STRAIN_TENSOR)
 KRATOS_CREATE_VARIABLE(Vector, CURVE_FITTING_PARAMETERS)
 KRATOS_CREATE_VARIABLE(Vector, PLASTIC_STRAIN_INDICATORS)
 KRATOS_CREATE_VARIABLE(double, EQUIVALENT_PLASTIC_STRAIN)
+KRATOS_CREATE_VARIABLE(Vector, KINEMATIC_PLASTICITY_PARAMETERS)
```

Comment:
Just one thing, be sure to define what each parameter means

Reply:
yes! there are 3 different models in kinematic plasticity, one model requires 1 param, the 2nd requires 2 and the more complex, 3. I'll describe it in the wiki but there are checks that gives an error if the size of the param vector is not the minimun required..